### ISSUE TITLE
convolution_3d.hpp problem with OpenMP when using VS

### ISSUE THREAD

**Issue by nenadpol** [PointCloudLibrary/pcl#1499]
Issue description:

error C3016: 'point_idx' : index variable in OpenMP 'for' statement must have signed integral type	............convolution_3d.hpp	

code in line 243 should be changed to: 
for (int point_idx = 0; point_idx < surface_->size (); ++point_idx) 

This is a known limitation in OpenMP 2.0 supported by Visual Studio 2012. OpenMP 3.0 accomodates the unsigned int in 'for' loop but not yet supported by Visual Studio.

http://www.pcl-users.org/convolution-3d-hpp-problem-with-OpenMP-when-using-VS-td4040580.html

**Comment by UnaNancyOwen** [comment]
I think that it should be fixes in accordance with the other source code.
In PCL, It seems that using cast to signed integer from unsigned integer in many cases.

[convolution_3d.hpp#L243](https://github.com/PointCloudLibrary/pcl/blob/master/filters/include/pcl/filters/impl/convolution_3d.hpp#L243)
```cpp
for (int point_idx = 0; point_idx < static_cast<int> (surface_->size ()); ++point_idx)
```

Do you have any good ideas?

**Comment by UnaNancyOwen** [comment]
I sent a pull request #1527.

**Comment by taketwo** [comment]
Fixed via  #1527.
